Wordle.Api/Wordle.Api/Controllers/WordController.cs

```diff
@@ -35,22 +33,27 @@ public async Task<Word> AddWord(string newWord, bool isCommon)
             return await _wordService.AddWordAsync(newWord, isCommon);
         }
 
+        [HttpDelete("Delete")]
+        public async Task<Word> DeleteWord(string word)
```

Comment:
you need to add a policy autherization check for some of these, create a new policy checking for Master and over 21 status

Reply:
for create and delete